docs/reference/pc_repository.md

```diff
@@ -0,0 +1,756 @@
+				}
+			]
+		}
+	]
+}
+```
+
+
+## `releases`
+
+All packages must have one or more `releases`.
+Releases reference tags named in accordance with [semver](https://semver.org).
+The value for `tags` can be simply `true` or a string prefix.
+The latter is specifically useful when targeting older versions of ST
+with a specific range of tags.
+
+Note that previously releases based on branches instead of tags was supported,
+but this has been deprecated.
+
+
+### `sublime_text`
+
+A release MUST contain a "sublime_text" version selector.
+Use `*` for all versions,
+or "<3000", "3500-4000", etc.
+to compare against specific build numbers of Sublime Text.
+Valid selectors are:
+
+- `<{build}`
+- `<={build}`
+- `>{build}`
+- `>={build}`
+- `{min_build} - {max_build}`
+
+In the example below, the entry with "sublime_text": "<3000" will match tags like "st2-1.0.0", "st2-1.1.0".  
+The release with "sublime_text": "3000 - 3999" will match tags like "st3-1.0.0", "st3-1.1.0".  
+The release with "sublime_text": ">=4000" will match tags like "st4-1.0.0", "st4-1.1.0".
+
+
+```json
+{
+	"details": "https://github.com/wbond/sublime_alignment",
+	"releases": [
+		{
+			"sublime_text": "<3000",
+			"tags": "st2-"
+		},
+		{
+			"sublime_text": "3000 - 3999",
+			"tags": "st3-"
+		},
+		{
+			"sublime_text": ">=4000",
+			"tags": "st4-"
+		}
+	]
+}
+```
+
+
+### `platforms`
+
+The `platforms` key allows specifying
+what platform(s) the release is valid for.
+Only the latest version for any given platform will be shown to the user.
+It allows specifying a single platform, or a list of platforms.
+Valid platform identifiers include:
+
+- \*
+- windows
+- osx
+- linux
+
+An architecture suffix (`-x32`, `-x64`) is supported
+but no longer relevant in today's world.
+
+Note that this is an optional property:
+if your releases supports all platforms you do not specify `"*"`,
+but instead omit the property completely.
+
+```json
+{
+	"details": "https://github.com/wbond/sublime_alignment",
+	"releases": [
+		{
+			"platforms": ["osx", "linux"],
+			"sublime_text": "*",
+			"tags": "posix-"
+		},
+		{
+			"platforms": "windows",
+			"sublime_text": "*",
+			"tags": "win32-"
+		}
+	]
+}
+```
+
+
+### `base`
+
+If for some reason one of the releases is from a different repository
+than the top-level "details" key,
+a "base" key may be specified in the release.
+This repository will then be used for the matching tags.
+
+```json
+{
+	"details": "https://github.com/wbond/sublime_alignment",
+	"releases": [
+		{
+			"base": "https://github.com/wbond/sublime_alignment",
+			"sublime_text": "<3000",
+			"tags": true
+		},
+		{
+			"sublime_text": ">=3000",
+			"tags": true
+		}
+	]
+}
+```
+
+### `version`, `url`, `date`
+
+For manually created release zips files,
+each individual "version" needs to be specified with
+a "url" to the zip, and a "date".
+Because this requires an update of the repository file for each release
+it is not allowed in the default repository.
```

Comment:
```suggestion
For manually created release zip files,
each individual version" needs to be specified with
a "url" to the zip, and a "date".
Because this requires an update of the repository file for each release,
it is not allowed in the default repository.
```